src/transaction/log_replication.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include "btree_load.h"
 #include "log_impl.h"
+#include "log_reader.hpp"
```

Comment:
is this needed as long as `log_reader` is included in `log_recovery_redo_parallel` header? or is it just for explicitness?

Reply:
it is not necessary, just for explicitness